examples/webgl_loader_collada_kinematics.html

```diff
@@ -120,14 +120,10 @@
 
 				// Lights
 
-				var directionalLight = new THREE.HemisphereLight( 0xffeeee, 0x111122 );
-				directionalLight.position.x = Math.random() - 0.5;
```

Comment:
The ground is normally horizontal, so a reasonable default direction is straight up, which corresponds to the hemisphere defined by the sky. Since `HemisphereLight.position` is actually a direction, yes, we could change the default to `( 0, 1, 0 )` -- a unit vector pointing up. Since the new `hemisphereLight` has no specular component, `( 0, 1, 0 )` is the only direction that makes sense to me -- other than `( 0, 0, 1 ) `, for users who prefer the z-axis as up.